python/rmm/tests/test_rmm.py

```diff
@@ -914,3 +915,39 @@ def test_rmm_device_buffer_copy(cuda_ary, make_copy):
     result = db_copy.copy_to_host()
 
     np.testing.assert_equal(expected, result)
+
+
+@pytest.fixture
+def torch_allocator():
+    try:
+        from torch.cuda.memory import change_current_allocator
+    except ImportError:
+        pytest.skip("pytorch pluggable allocator not available")
+
+    try:
+        change_current_allocator(rmm.rmm_torch_allocator)
+    except RuntimeError:
+        pass
+
+
+def test_rmm_torch_allocator(torch_allocator, stats_mr):
+    import torch
```

Comment:
Is `torch` already an RMM test dependency?

Reply:
Good question -- I am not excited about the number of pytorch testing dependencies we have across RAPIDS, because the packages have plenty of conflicts (newer `pytorch::pytorch` packages are built against `nvidia::cuda-toolkit` instead of `conda-forge::cudatoolkit`, poor ARM support, ...)

Reply:
You can see how we handle this in cudf right now here: https://github.com/rapidsai/cudf/blob/ec45c18c0f88acbf8f518a3ac87c3181da93ee2c/dependencies.yaml#L293-L295

We only run pytorch tests on x86_64, and we have to pin to `<1.12.0`.

Reply:
> and we have to pin to `<1.12.0`

Then the new tests would fail because this feature is not even released (it's only available in nightly AFAIR).

Reply:
It is not -- are we OK with not testing on CI for now? 

I noticed that CuPy is also not currently a test dependency AFAICT, and yet we have tests that `importorskip("cupy")`

Reply:
Would it make sense to put these in say `test_rmm_pytorch.py` and use `importorskip` there?

Reply:
Thanks, @jakirkham - fantastic suggestion. I've gone ahead and done so.